package.json

```diff
@@ -24,7 +24,8 @@
     "commitmsg": "commitlint -e",
     "start": "ember server",
     "semantic-release": "semantic-release pre && npm publish && semantic-release post",
-    "test": "ember try:each"
+    "test": "ember try:each",
+    "inspect": "node --inspect --debug-brk node_modules/.bin/ember s"
```

Comment:
@alexdiliberto Haven't seen this one in action yet. This enables dev tools? Any good articles?

Reply:
@crodriguez1a Yea it's really nice if you want to debug node in the browser using chrome dev tools. It's what I used to make the `index.js` updates.  Here: https://hackernoon.com/debug-node-js-with-chrome-devtools-aca7cf83af6b